prac_06/guitars.py

```diff
@@ -0,0 +1,26 @@
+from prac_06.guitar import Guitar
+
+
+def main():
+    guitars = []
+
+    print("My Guitars!")
+    name = input("Name: ")
+    while name != "":
+        year = int(input("Year: "))
+        cost = float(input("Cost: $"))
+        new_guitar = Guitar(name, year, cost)
+        guitars.append(new_guitar)
+        print(new_guitar, " added.")
+        name = input("Name: ")
+
+    print("These are my guitars:")
+    for i, guitar in enumerate(guitars, 1):
+        vintage_string = ""
+        if guitar.is_vintage():
+            vintage_string = "(vintage)"
```

Comment:
You could make this a one liner if you wanted to:
`vintage_string = " (vintage)" if guitar.is_vintage() else ""`

Reply:
That's a cool idea, thanks :)